components/elm/src/biogeophys/SoilHydrologyType.F90

```diff
@@ -569,7 +569,12 @@ subroutine InitCold(this, bounds)
          endif
 
          ! set decay factor
-         this%hkdepth_col(c) = 1._r8/2.5_r8
```

Comment:
Why changes it?

Reply:
This is to allow ELM to read `fdrain` from surface data. If `fdrain` is not showing up in surface data, the default `fdrain=2.5` will be used.

Reply:
Do you know why fdrain was not read from surface data in the current version?

Reply:
I don't know the reason. So, I only let ELM to read `fdrain` from surface data when land river two-way coupling is turned on.